hack/tools/tilt-prepare/main.go

```diff
@@ -285,6 +286,19 @@ func tiltResources(ctx context.Context, ts *tiltSettings) error {
 		tasks[providerName] = providerTask(providerName, fmt.Sprintf("%s/config/default", path), ts)
 	}
 
+	// Add an addon task for each name/container_name/binary_name/path defined using the --addon flag.
```

Comment:
Could we also read this from tilt-settings.yaml + test-extension is always included?